src/orchestration/Orchestration.ts

```diff
@@ -159,6 +160,7 @@ export type Config = {
     sessionEventLimit: number;
     sessionLengthSeconds: number;
     sessionSampleRate: number;
+    spaActivityTimeoutLimit?: number;
```

Comment:
issue: `SPA activity` is not common terminology. Let's think of something clearer. "Route change" is common terminology amongst front-end frameworks. Consider `routeChangeTimeout`.

Similarly, I recommend replacing other instances of the term `activity` in the code and documentation.